src/routes/UserRouter.ts

```diff
@@ -30,17 +62,33 @@ export class UsersRouter {
       .catch((error) => {
         // console.log("ERROR:", error.message);
       });
-    // res.send(Users);
-    db.any('SELECT * FROM public."Users"')
+    db.any('SELECT row_to_json(t) FROM (SELECT u.*, json_agg(p) FROM "Users" u JOIN public."Permissions" p ON u."Role" = p."Role" GROUP BY u."User_Id") t	')
       .then((data) => {
         res.status(200)
           .json({
             data,
             message: 'Retrieved ALL Users',
             status: 'success'
           });
-        // console.log(data);
+
       })
       .catch((err) => next(err));
   }
+  /**
+   * Create new user
+   */
+  public static createUser(req: Request, res: Response, next: NextFunction) {
+    // = parseInt(req.body.users, 10);
+    db.none('INSERT INTO public."Users"("Username", "Email", "Password", "Role") VALUES ({$username}, {$email}, {$password},{$role})', req.body)
+      .then(() => {
+        res.status(200)
```

Comment:
Please use status [201 (Created)](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success) when rows are inserted.

Reply:
I will update this